src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -87,11 +87,11 @@ def load_command_table(self, _):
         g.command('get-upgrades', 'get_upgrade_profile', table_transformer=aks_upgrades_table_format)
         g.custom_command('install-cli', 'k8s_install_cli', client_factory=None)
         g.custom_command('list', 'aks_list', table_transformer=aks_list_table_format)
-        g.custom_command('remove-dev-spaces', 'aks_remove_dev_spaces')
+        g.custom_command('remove-dev-spaces', 'aks_remove_dev_spaces', deprecate_info=g.deprecate())
         g.custom_command('scale', 'aks_scale', supports_no_wait=True)
         g.custom_show_command('show', 'aks_show', table_transformer=aks_show_table_format)
         g.custom_command('upgrade', 'aks_upgrade', supports_no_wait=True)
-        g.custom_command('use-dev-spaces', 'aks_use_dev_spaces')
+        g.custom_command('use-dev-spaces', 'aks_use_dev_spaces', deprecate_info=g.deprecate())
```

Comment:
Hi @amsoedal , the deprecate object is define [here](https://github.com/microsoft/knack/blob/dc7b7d857ce3c11b2a3155b81f2470b3b579e7c9/knack/deprecation.py#L48), you can read and see how to support your expected behavior. There are also lots of examples in CLI code, see [here](https://github.com/Azure/azure-cli/blob/1a41c62162b228c03e5b2bf7d681735d88f96b88/src/azure-cli/azure/cli/command_modules/keyvault/commands.py#L231) is you only want to add a warning message.

Reply:
Hi @arrownj , thanks for pointing me to that documentation.  I'd like to use the default message, the way the PR is right now.  Can this PR be merged?